src/components/Tooltip.js

```diff
@@ -9,11 +9,22 @@ class Tooltip extends Component {
         super(props);
     }
 
+    /**
+     * @author Xiaheng Chen
+     * 
+     * @returns {string}
+     */
+    tooltipPositionClass() {
```

Comment:
This returns a computed property. So I think a better naming convention would be to `getPositionClass`. We don't need to explicitly tell `getTooltipPositionClass` because it is inside the `Tooltip` class.